components/server/src/server.rs

```diff
@@ -1257,3 +1285,97 @@ impl<R: RaftEngine> EngineMetricsManager<R> {
         }
     }
 }
+
+pub struct EnginesResourceInfo {
+    kv_engine: RocksEngine,
+    raft_engine: Option<RocksEngine>,
+    normalized_num_mem_tables: MovingMaxU32,
```

Comment:
why these statistics are collected in moving max instead of moving average

Reply:
Originally use `max` to avoid hitting write stalls. Updated to use `max(avg, current)`.